requirements.txt

```diff
@@ -1,22 +1,35 @@
 black
+cloudpickle
 click
+dash
+dask
+datashader
+datetime
+dash_bootstrap_components
 email-validator
+fsspec
 flake8
 Flask
 flask-admin
+flask_bootstrap
 Flask-Login
 flask-migrate
 Flask-Mail
 flask_mail
 Flask-SQLAlchemy
 Flask-WTF
+gunicorn
 itsdangerous
 Jinja2
 MarkupSafe
 mysqlclient
+numpy
+pandas
 pycountry
 pytest
+pathlib
 python-dotenv
 SQLAlchemy
```

Comment:
please set "SQLAlchemy" and "Flask-SQLAlchemy" to version 1.3.23, because new version unstable